project/frontend/src/scripts/recommendation_script.js

```diff
@@ -1,8 +1,8 @@
 export default function getRecommendation(preferences, callback) {
```

Comment:
This isn't important but it might make this function a lot cleaner to use async/await instead of chaining `.then()`s together. All that you'd have to do is mark the function async like `async function getRecommendation(...` and then whenever a `.then()` callback is used. You could instead do something like:

```
const responses = await Promise.all(promises);
const data = await Promise.all(responses.map((response) => response.json()));

// do stuff with data

const response = await fetch('/api/recommendation', ...);
const data = await response.text();
// do stuff with data
```

Reply:
I like this idea-- I think I'll come back to it in a separate PR (maybe after our demo tomorrow)